challenge.py

```diff
@@ -16,13 +18,31 @@ def run():
     division_by_18 = make_division_by(18)
     print(division_by_18(54))  # The expected output is 3
 
+    unittest.main()
+
 
 if __name__ == '__main__':
     import unittest
 
     class ClosureSuite(unittest.TestCase):
+
+        def setUp (self):
+            self.value_LCM = 90
```

Comment:
Este valor no se requiere en varios test, lo puedes definir directamente en el único test.

Reply:
Tests can be numerous, and their set-up can be repetitive. Luckily, we can factor out set-up code by implementing a method called setUp(), which the testing framework will automatically call for every single test we run:

Reply:
https://docs.python.org/3/library/unittest.html#organizing-test-code